presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -23,6 +23,39 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
+
+Connection security
+------------------
+
+The JDBC driver and connector automatically use Transport Layer Security (TLS) encryption and certificate validation. This requires a suitable TLS certificate configured on your SQL Server database host.
+
+To disable encryption in the connection string, use the encrypt property:
+
+.. code-block:: none
+
+    connection-url=jdbc:sqlserver://<host>:<port>;databaseName=<databaseName>;encrypt=false;
+
+Further parameters like trustServerCertificate, hostNameInCertificate, trustStore, and trustStorePassword can be configured as below
```

Comment:
```suggestion
```
This sentence repeating the property names from the immediately following table doesn't add to the user's understanding, so I think we can delete it entirely. Is that reasonable to you?

Reply:
But how would users know the purpose of the following properties

Reply:
@steveburnett I have changed can you review.

Reply:
> But how would users know the purpose of the following properties

Good point! I understand. I suggested a revision in the new review, let me know what you think. Thanks!